src/WebJobs.Script.WebHost/Management/FunctionsSyncManager.cs

```diff
@@ -313,23 +313,26 @@ public async Task<SyncTriggersPayload> GetSyncTriggersPayload()
             var triggersArray = new JArray(triggers);
             int count = triggersArray.Count;
 
+            // Form the base minimal result
+            string hostId = await _hostIdProvider.GetHostIdAsync(CancellationToken.None);
```

Comment:
We're now ALWAYs sending the object format on syncs. Previously when cache was disabled or we were otherwise returning a "minmal" format, we were only sending an array. FE/ScaleController handles both formats of course so this isn't a problem.